src/components/AutoSuggestion/SingleLineCodeEditor.tsx

```diff
@@ -1,38 +1,39 @@
-import "codemirror/addon/display/placeholder.js"
+import "codemirror/addon/display/placeholder.js";
 import "codemirror/mode/sparql/sparql.js";
 import React from "react";
-import {UnControlled as UnControlledEditor} from "react-codemirror2";
-import {Classes as BlueprintClassNames} from "@blueprintjs/core";
-import {Editor as CodeMirrorEditor} from "codemirror";
-import {CLASSPREFIX as eccgui} from "../../configuration/constants";
+import { UnControlled as UnControlledEditor } from "react-codemirror2";
+import { Classes as BlueprintClassNames } from "@blueprintjs/core";
+import { Editor as CodeMirrorEditor, EditorChange } from "codemirror";
+import { CLASSPREFIX as eccgui } from "../../configuration/constants";
 
 export interface IRange {
-  from: number
-  to: number
+    from: number;
+    to: number;
 }
 
 export interface SingleLineCodeEditorProps {
```

Comment:
The component should be renamed to reflect the multi-line option, e.g. `CodeEditor`.